src/fisher.jl

```diff
@@ -97,6 +158,18 @@ function pvalue_both_minlike(x::FisherExactTest, ω::Float64=1.0)
 end
 
 # confidence interval by inversion of p-value
+"""
+    confint(x::FisherExactTest, alpha::Float64=0.05; tail=:both, method=:central)
+
+Compute a confidence interval with coverage 1 - `alpha` by inverting the `:central` p-value.
```

Comment:
Would be nice to repeat the definition of different methods here. AFAIK there's no need to mention the p-value here at all.